build/buildimage/differ.go

```diff
@@ -84,6 +86,112 @@ func NewRegistryDiffer(platform v1.Platform) Differ {
 	}
 }
 
+func parseApplicationImages(srcPath string) ([]string, error) {
+	// TODO this should be global
+	applicationPath := filepath.Join(srcPath, "application/apps")
```

Comment:
Actually, we need to have a `rootfs` pkg or manager to provide the application path relative to rootfs.(Make it TODO?)

Reply:
In runtime and it sub module, It provide by private func such as: getRootfs or something. It duplicate with other module in runtime. I think have a public pkg to manage path is a good suggestion!